core/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala

```diff
@@ -20,18 +22,24 @@ object ExpandImportSelectors extends Rewrite {
                 }
             }.flatten
             val patchStr = expandedImport.mkString("\n")
-            Patch(`import`.parent.get.tokens.head,
-                  `import`.parent.get.tokens.last,
-                  patchStr)
+
+            //TODO move up this side effect
+            `import`.parent.get.tokens.tail.foreach { tok =>
```

Comment:
It would be nice to add helper utilities for common operations like this. For example, `Patch.remove(Tree/Tokens)`

We can leave that for a separate PR!